larreco/ClusterFinder/Cluster3D_module.cc

```diff
@@ -480,6 +480,7 @@ namespace lar_cluster3d {
     /**
      *   Algorithm parameters
      */
+    bool m_onlyMakSpacePoints;    ///< If true we don't do the full cluster 3D processing
```

Comment:
Not sure is `Mak` was a character limit thing or a typo but, for the sake of one character, this could probably be `Make` instead?